app/template.ejs

```diff
@@ -210,7 +210,7 @@
         </div>
     </div>
 
-    <script src="/app/libs/outdated-browser/outdatedbrowser/outdatedbrowser.min.js"></script>
+    <script src="/app/libs/outdated-browser/outdatedbrowser/outdatedbrowser.min.js?v=<%=gitVersion%>"></script>
```

Comment:
why reload this lib on new accounts versions?

Reply:
My thought was if it was updated (version# in bower), the user would still get the cached version.  Am I incorrect?

Reply:
Downside, they download everything on any change.  Upside we have control over it so we can force it if needed.  I was not sure